pr_agent/algo/utils.py

```diff
@@ -89,8 +89,8 @@ def try_fix_json(review, max_iter=10, code_suggestions=False):
             data = {}
     return data
 
+
 def fix_json_escape_char(json_message=None):
-    result = None
     try:
         result = json.loads(json_message)
     except Exception as e:
```

Comment:
Consider handling the specific JSONDecodeError exception instead of a general Exception. This would make the error handling more precise and avoid catching unintended exceptions. [medium]